src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -152,6 +152,7 @@ message VoidPaymentRequest {
   }
```

Comment:
I know you don't want anymore renames, but we should _really_ rename the pay/details response field `idempotency_uuid` to `payment_id` and then rename `order_id` field in the other requests (cancel/refund) to `payment_id` 😄